pkg/scripts/download.go

```diff
@@ -44,14 +44,14 @@ func NewDownloader(proxyConfig *api.ProxyConfig) Downloader {
 // Returns information a boolean flag informing whether any download/copy operation
 // happened and an error if there was a problem during that operation
 func (d *downloader) Download(url *url.URL, targetFile string) (*api.SourceInfo, error) {
-	schemeReader := d.schemeReaders[url.Scheme]
+	r := d.schemeReaders[url.Scheme]
```

Comment:
> it's not really shadowing anything since they are two different syntactical elements.

@bparees @php-coder look what I came across today from golang-nuts:
https://groups.google.com/d/topic/golang-nuts/QOH-9wywB38/discussion

- Variables and types live in the same namespace, so one can shadow the
other.
- Types can also shadow variables.
- And variables can shadow packages.